client/messages/fr.json

```
@@ -1,4 +1,388 @@
 {
+  "cQyKR1": "Points d'accès",
```

Biome (2.4.13)

[error] The key cQyKR1 was already declared.

(lint/suspicious/noDuplicateObjectKeys)